util/fhir.ts

```diff
@@ -160,7 +160,8 @@ export function createFHIRResourceString(
   let codeData;
   if (primaryCodeType === 'FHIR.CodeableConcept') {
     codeData = {
-      coding: coding
+      // Need to add coding as an array for codeable concept
+      coding: [coding]
```

Comment:
Nitpick but can you make the type of `codeData` above `fhir4.CodeableConcept | fhir4.Coding | string | undefined | null`? That should prevent these kinds of errors in this function in the future, and it should still work